src/coreclr/jit/fgbasic.cpp

```diff
@@ -1608,10 +1637,8 @@ void Compiler::fgFindJumpTargets(const BYTE* codeAddr, IL_OFFSET codeSize, Fixed
                     }
                 }
 
-                if ((codeAddr < codeEndp - sz) && (OPCODE)getU1LittleEndian(codeAddr + sz) == CEE_RET)
+                if ((codeAddr < codeEndp - sz) && doesMethodLookLikeWrapper(codeAddr + sz, codeEndp))
```

Comment:
Given it's called only here, it only supports
```
call
ldloc
box
ret
```
?

is this some specific pattern we need to support?

Reply:
nope. it's just that `doesMethodLookLikeWrapper` can return `true` for a single `ret` so that it can be reused here.